README.adoc

```diff
@@ -1162,7 +1162,7 @@ end
 # same table you had during the creation of the migration.
 # In future if you override the `Product` class and change the `table_name`,
 # it won't break the migration or cause serious data corruption.
-class MigrationProduct < ActiveRecord::Base
```

Comment:
I'm on the fence with changing it here, and the reason is https://github.com/rubocop/rails-style-guide/issues/284.
Wondering what enhancements could go into `ApplicationRecord` that would be useful in migrations.
And since those enhancements are a subject to change over time, running historical migrations can be affected by the same issue as with renaming/removing models.
I'd keep this guideline intact, maybe with a clarification.

Reply:
I've put this one back to `ActiveRecord::Base`.